WalletWasabi.Packager/Program.cs

```diff
@@ -306,6 +310,12 @@ private static void Publish()
 				Console.WriteLine($"Deleted {libraryBinReleaseDirectory}");
 			}
 
+			var deterministicFilenameTag = IsContinuousDelivery ? $"{DateTimeOffset.UtcNow:ddMMyyyy}" : VersionPrefix;
+			var deliveryPath = IsContinuousDelivery ? Path.Combine(BinDistDirectory, "cdelivery") : BinDistDirectory;
```

Comment:
Is `cdelivery` good name? Why not use `continuousDelivery` to make it clear.

Reply:
I am not able to type `continuous` correctly. No matter how hard I tried I always make a mistake.

Reply:
Why not to write a Powershell script that calls it for you with proper name then? (Powershell 7 is multiplatorm beast so that it can be even used on Linux too.)

Reply:
cdelivery is good here, easy to use, and cannot be misspelled.